examples/rapthor.parset

```diff
@@ -142,13 +142,15 @@ input_skymodel = /data/skymodel.txt
 # batch_system = singleMachine
 
 # For batch_system = slurm, the maximum number of nodes of the cluster to use at
-# once can be specified with the max_nodes option (default = 12), and the number
-# of processors per task to request from SLURM can be specified with the
-# cpus_per_task option (default = 0 = all). By setting the cpus_per_task value to the
-# number of processors per node, one can ensure that each task gets the entire
-# node to itself, which is the recommended way of running Rapthor
+# once can be specified with the max_nodes option (default = 12), the number of
+# processors and amount of memory per node to request from SLURM can be
+# specified with the cpus_per_task (default = 0 = all) and mem_per_node_gb
+# options (default = 0 = all). By setting the cpus_per_task value to the number of
+# processors per node, one can ensure that each task gets the entire node to
+# itself, which is the recommended way of running Rapthor
```

Comment:
This only works if you have a (more or less) homogeneous cluster. What to do if you have nodes with different numbers of CPUs?

Reply:
Hmm, good question. I expect most such clusters would have different partitions for the different node types, but specifying a partition is not supported yet in Rapthor either.